README.md

```diff
@@ -18,57 +18,78 @@ LiteFarm is comprised of three applications which all reside in this monorepo.
 
 ## Preliminaries 
 
-1. Install [node.js](https://nodejs.org/en/download/package-manager/) if you do not already have it.
-2. If you do not have pnpm installed, run `npm install -g pnpm` in a terminal.
-3. Use the `git clone` command to clone this repository to your computer.
-4. In a terminal, navigate to the root folder of the repo and run `npm install`.
-5. Navigate to the `packages/api` folder, and run `npm install`.
-6. Navigate to the `packages/webapp` folder, and run `pnpm install`.
-
-
-## Configuring the applications
-
-The applications are configured with environment variables stored in `.env` files. Configuration information includes secrets like API keys, so the `.env` files should never be added to source control. This repository does contain `.env.default` files for api and webapp. These files contain all necessary environment variables, but for sensitive ones the values are redacted. Contact smattingly@litefarm.org for assistance.
+1. Check to see if you have Node.js installed. On a Mac use the command `node-v` in terminal. If it is installed, the version in use will be reported in the terminal. If not, install it from [node.js](https://nodejs.org/en/download/package-manager/).
+2. Check to see if you have pnpm installed. On a Mac use the command `pnpm -v`. If it is installed, the version will be reported. If you do not have it installed, run `npm install -g pnpm` in a terminal. 
+3. Check to see if you have NVM installed. On a Mac use the command `nvm -v`. If you do not have NVM (Node Version Manager) installed, install it using these instructions: [NVM](https://www.loginradius.com/blog/engineering/run-multiple-nodejs-version-on-the-same-machine/)
+4. Clone the repository from Github to your computer. On a Mac, in a Terminal window use the command `gh repo clone LiteFarmOrg/LiteFarm`.
```

Comment:
This command relies on the github cli - we should instead use the default git clone command in the guide as it isn't necessary for someone to use the github cli to clone the app

Reply:
Oh, simple enough. I'll change it to the regular git command using https.